_posts/2017-10-16-flasheando-custom-firmware-router.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: Flasheando firmware customs a routers para agregar funcionalidades y mejorar seguridad.
+description: Guía básica de porque y como flashear firmware customs a routers hogareños.
+authors: alexdaciuk
+tags: lugfi unix seguridad
```

Comment:
El tag *lugfi* es redundante, ya que se publica en el sitio de lugfi.

Reply:
![face-you-make-robert-downey-jr](https://user-images.githubusercontent.com/2780694/31737998-a3bb8b80-b41f-11e7-8650-3936fb5e297c.jpg)